train/point1_train.prototxt

```diff
@@ -0,0 +1,941 @@
+  top: "conv1"
+  convolution_param {
+    num_output: 128
+    kernel_size: 2
+    stride: 2
+    weight_filler {
+      type: "gaussian"
+      std: 0.1
+    }
+  }
+}
+layer {
+  name: "conv1_bn"
+  type: "BatchNorm"
+  bottom: "conv1"
+  top: "conv1_bn"
+}
+layer {
+  name: "relu1"
+  type: "ReLU"
+  bottom: "conv1_bn"
+  top: "conv1_bn"
+}
+layer {
+  name: "conv2"
+  type: "Convolution"
+  bottom: "conv1_bn"
+  top: "conv2"
+  convolution_param {
+    num_output: 128
+    kernel_size: 3
+    weight_filler {
+      type: "gaussian"
+      std: 0.1
+    }
+  }
+}
+layer {
+  name: "conv2_bn"
+  type: "BatchNorm"
+  bottom: "conv2"
+  top: "conv2_bn"
+}
+layer {
+  name: "relu2"
+  type: "ReLU"
+  bottom: "conv2_bn"
+  top: "conv2_bn"
+}
+layer {
+  name: "conv3"
+  type: "Convolution"
+  bottom: "conv2_bn"
+  top: "conv3"
+  convolution_param {
+    num_output: 64
+    kernel_size: 3
+    weight_filler {
+      type: "gaussian"
+      std: 0.1
+    }
+  }
+}
+layer {
+  name: "conv3_bn"
+  type: "BatchNorm"
+  bottom: "conv3"
+  top: "conv3_bn"
+}
+layer {
+  name: "relu3"
+  type: "ReLU"
+  bottom: "conv3_bn"
+  top: "conv3_bn"
+}
+layer {
+  name: "conv4"
+  type: "Convolution"
+  bottom: "conv3_bn"
+  top: "conv4"
+  convolution_param {
+    num_output: 32
+    kernel_size: 3
+    weight_filler {
+      type: "gaussian"
+      std: 0.1
+    }
+  }
+}
+layer {
+  name: "conv4_bn"
+  type: "BatchNorm"
+  bottom: "conv4"
+  top: "conv4_bn"
+}
+layer {
+  name: "relu4"
+  type: "ReLU"
+  bottom: "conv4_bn"
+  top: "conv4_bn"
+}
+layer {
+  name: "conv5"
+  type: "Convolution"
+  bottom: "conv4_bn"
+  top: "conv5"
+  convolution_param {
+    num_output: 16
+    kernel_size: 3
+    weight_filler {
+      type: "gaussian"
+      std: 0.1
+    }
+  }
+}
+layer {
+  name: "conv5_bn"
+  type: "BatchNorm"
+  bottom: "conv5"
+  top: "conv5_bn"
+}
+layer {
+  name: "relu5"
+  type: "ReLU"
+  bottom: "conv5_bn"
+  top: "conv5_bn"
+}
+layer {
+  name: "conv6"
+  type: "Convolution"
+  bottom: "conv5_bn"
+  top: "conv6"
+  convolution_param {
+    num_output: 16
+    kernel_size: 3
+    weight_filler {
+      type: "gaussian"
+      std: 0.1
+    }
+  }
+}
+layer {
+  name: "conv6_bn"
+  type: "BatchNorm"
+  bottom: "conv6"
+  top: "conv6_bn"
+}
+layer {
+  name: "relu6"
+  type: "ReLU"
+  bottom: "conv6_bn"
+  top: "conv6_bn"
+}
+layer {
+  name: "conv7"
+  type: "Convolution"
+  bottom: "conv6_bn"
+  top: "conv7"
+  convolution_param {
+    num_output: 16
+    kernel_size: 3
+    weight_filler {
+      type: "gaussian"
+      std: 0.1
+    }
+  }
+}
+layer {
+  name: "conv7_bn"
+  type: "BatchNorm"
+  bottom: "conv7"
+  top: "conv7_bn"
+}
+layer {
+  name: "relu7"
+  type: "ReLU"
+  bottom: "conv7_bn"
+  top: "conv7_bn"
+}
+layer {
+  name: "conv8"
+  type: "Convolution"
+  bottom: "conv7_bn"
+  top: "conv8"
+  convolution_param {
+    num_output: 16
+    kernel_size: 3
+    weight_filler {
+      type: "gaussian"
+      std: 0.1
+    }
+  }
+}
```

Comment:
I can add this, I am also trying to fix the other errors. Not sure, but I think we may have to convert images to leveldb/lmdb format (using create_imagenet.sh*)

* http://caffe.berkeleyvision.org/gathered/examples/imagenet.html